hazelcast/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -106,6 +106,13 @@
             "  - Change XML/YAML configuration property: Set hazelcast.jet.enabled to true\n" +
             "  - Add system property or environment variable: -Dhz.jet.enabled=true or HZ_JET_ENABLED=true";
 
+    public static final String JET_RESOURCE_UPLOAD_DISABLED_MESSAGE = "The JobConfig contains resources to upload, " +
```

Comment:
The first sentence should tell developers what is wrong in the most simple terms. We can be more specific when we suggest fixes.

```suggestion
    public static final String JET_RESOURCE_UPLOAD_DISABLED_MESSAGE = "A job is trying to upload resources to the cluster, but it does not have permission." +
```